qa/suites/upgrade/quincy-p2p/quincy-p2p-stress-split/0-cluster/start.yaml

```diff
@@ -0,0 +1,33 @@
+meta:
+- desc: |
+   Run ceph on two nodes,
+   with a separate client-only node.
+   Use xfs beneath the osds.
+overrides:
+  ceph:
+    fs: xfs
+    log-ignorelist:
+      - overall HEALTH_
+      - \(MON_DOWN\)
+      - \(MGR_DOWN\)
```

Comment:
Just curious, why is there such a big difference between ignorelist here and in `qa/suites/upgrade/quincy-p2p/quincy-p2p-parallel/point-to-point-upgrade.yaml`?

Reply:
@idryomov I see-- got it.Is I see that @chrisphoffman's PR targets pacific-x and quincy-x; are there any plans to modify pacific-p2p tests?

Also, I added the appropriate ignorelists to two relevant rbd yaml files in this PR and have scheduled new runs. So we'll see how those turn out.

Reply:
FWIW I don't think `rbd/test_librbd_python.sh` needs them.

Reply:
Removed the extra whitelist.